firstvoices/backend/resources/utils/import_export_widgets.py

```diff
@@ -68,3 +71,13 @@ def clean(self, value, row=None, **kwargs):
                 **{self.field: DUMMY_USER_EMAIL}, defaults={"email": DUMMY_USER_EMAIL}
             )
             return dummy_user
+
+
+class AudienceBooleanFieldWidget(Widget):
```

Comment:
Can also be renamed to `ToggleBooleanFieldWidget` if its reused in more places than the Audience flags.

Reply:
Maybe ReverseBooleanFieldWidget since it uses the opposite value

Reply:
I'd consider naming this `InvertedBooleanWidget` or similar to indicate the value flip. Since you're here it might also be valuable to add a `render` method to show how the db value would be exported to CSV (it would probably also need to be flipped).

Also, where user input is concerned we will need to accept a wider range of values, but that could be a separate ticket.

Reply:
Thank you for the suggestion. I have updated the name to `InvertedBooleanFieldWidget`. 
Also, I've added a ticket for the render method at https://firstvoices.atlassian.net/browse/FW-5842.